### PR TITLE
Close after importing keys from geth

### PR DESCRIPTION
**Pull request by svyatonik**
closes ethcore/parity#1918

### CODE REVIEW

**Comment by svyatonik** [comment]
[clabot:check]

File: parity/account.rs [hunk 1]
```diff
@@ -86,3 +94,17 @@ fn import(i: ImportAccounts) -> Result<String, String> {
 	}
 	Ok(format!("{}", imported))
 }
+
+fn import_geth(i: ImportFromGethAccounts) -> Result<String, String> {
+	use std::io::ErrorKind;
+	use ethcore::ethstore::Error;
+
+	let dir = Box::new(try!(keys_dir(i.to)));
+	let secret_store = EthStore::open(dir).unwrap();
```
Comment (rphmeier) [inline parity/account.rs]: maybe `try!(EthStore::open(dir).map_err(|e| format!("Error opening key store: {}", e)))`

File: parity/account.rs [hunk 2]
```diff
@@ -39,11 +40,18 @@ pub struct ImportAccounts {
 	pub to: String,
 }
 
+#[derive(Debug, PartialEq)]
+pub struct ImportFromGethAccounts {
+	pub testnet: bool,
+	pub to: String,
+}
```
Comment (rphmeier) [inline parity/account.rs]: documentation on this struct and fields would be a bonus :)

File: parity/account.rs (hunk 1 above)
Comment (svyatonik) [inline parity/account.rs]: Copy-pasted this line from other functions above (new, list) :) Just tried to mimic the code-style from previous contributors. Ok - will fix this for these functions too

**Comment by rphmeier** [comment]
LGTM, but haven't tested it yet.

**Comment by NikVolf** [comment]
```
admin@nikvolf ~/g/sv-test> cargo run -- account import ./ethstore/tests/res/geth_keystore/
    Finished debug [unoptimized + debuginfo] target(s) in 0.1 secs
     Running `target/debug/parity account import ./ethstore/tests/res/geth_keystore/`
3
```

just types `3`
or is it as designed?

**Comment by svyatonik** [comment]
This PR is not about --account import - it's about --import-geth-keys

**Comment by NikVolf** [comment]
tested import, great job

**Pull request merged**